compiler-rt/lib/asan/asan_interceptors.cpp

```diff
@@ -65,6 +65,15 @@ static inline uptr MaybeRealStrnlen(const char *s, uptr maxlen) {
   return internal_strnlen(s, maxlen);
 }
 
+static inline uptr MaybeRealWcsnlen(const wchar_t *s, uptr maxlen) {
+#  if SANITIZER_INTERCEPT_STRNLEN
```

Comment:
Is this supposed to be `SANITIZER_INTERCEPT_WCSLEN`? Or should there be a wide string equivalent?